engines/python/setup/djl_python/deepspeed.py

```diff
@@ -128,8 +128,6 @@ def initialize(self, properties: dict):
                 "max_seq_len": int(properties.get("max_tokens", 1024)),
                 "tokenizer": self.tokenizer
             }
-            if "output_formatter" in properties:
```

Comment:
Why we remove this? Should the value in `serving.properties` be the default value?

Reply:
This is setting output_formatter in kwargs to initialize DeepSpeedRollingBatch, but we moved output_formatter out of DeepSpeedRollingBatch to request level. So it's not needed.